onnxruntime/core/profile/profile.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/profile/profile.h"
+#include "core/common/common.h"
+#if !defined(NDEBUG) && defined(USE_CUDA)
+#include <nvToolsExt.h>
+#include <nvToolsExtCuda.h>
+#endif
+
+namespace onnxruntime {
+namespace profile {
+
+void NvtxRangeCreator::BeginImpl() {
+// enable only for debug builds because this function is for profiling only.
```

Comment:
>enable only for debug build [](start = 3, length = 27)

Could you add a new compiler flag like CUDA_PROFILE instead?

Reply:
Good idea!

---
In reply to: [428780166](https://github.com/microsoft/onnxruntime/pull/4013#discussion_r428780166) [](ancestors = 428780166)